### PR TITLE
Make it easy to plot training & validation loss on same figure

### PR DESCRIPTION
**Pull request by ashtonsix**
Record epochs to align plots.

plot showing training & validation loss on same figure

### CODE REVIEW

**Comment by ashtonsix** [comment]
Oh, & I just added times because it seemed like a good idea
